src/test/java/org/example/SearchField.java

```diff
@@ -1,5 +1,99 @@
 package org.example;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.nio.charset.Charset;
+import java.time.Duration;
+import java.util.List;
+import java.util.Random;
+
 public class SearchField {
+    private WebDriver webDriver;
+
+    @BeforeClass
+    private void setDriver() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+        webDriver = new ChromeDriver();
+    }
+
+    @Test
+    private void searchFieldWrite() {
+        openAmazonWebPage();
+        webDriver.manage().window().maximize();
+        String textTry= randomText();
+        WebElement searchBar = webDriver.findElement(By.id("twotabsearchtextbox"));
+        searchBar.sendKeys(textTry);
+
+        WebElement searchButton = webDriver.findElement(By.id("nav-search-submit-text"));
+        searchButton.click();
+
+        WebElement textErorr = webDriver.findElement(By.xpath("//*[@id=\"search\"]/div[1]/div[1]/div/span[3]/div[2]/div[1]/div/div/div/div[1]"));
+        Assert.assertEquals(String.format("No results for %s.", textTry), textErorr.getText());
+
+    }
+
+    private String randomText() {
```

Comment:
method names should contain verbs. maybe generateRandomText() or getRandomText() could be better